html/about.html

```diff
@@ -0,0 +1,88 @@
+                <li class="programs">
+                    <img src="/images/exhibitionIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Exhibition</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/forumIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Forum</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/workshopIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Workshop</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/networkingIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Network</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+            </ul>
+            <a class="join none"> Join LT Global Summit 2022 </a>
+            <p class="see-projects none1"><a>SEE THE WHOLE PROJECTS</a></p>
+        </section>
+
+        <!-- inject the speakers dynamically with javascript -->
+        <section class="speakers">
+            <h3 class="mp-title">Featured Speakers</h3>
+            <hr class="spk-hr" />
+            <div class="grid">
+
+            </div>
+        </section>
+
+        <section class="partners" id="sponsor">
+            <h3>Partners</h3>
+            <hr class="hr-about" />
+            <div class="partners-logo">
+                <img src="/images/mozilla.png" width="100" height="50" alt="mozilla" />
+                <img src="/images/google.png" width="100" height="50" alt="google" />
+                <img src="/images/naver.png" width="100" height="50" alt="naver" />
+                <img src="/images/daumkakao.jpg" width="100" height="50" alt="daumkakao" />
+                <img src="/images/airbnb.png" width="100" height="50" alt="airbnb" />
+            </div>
+        </section>
+    </section>
+    <script src="../js/mMenu.js"></script>
+    <script src="../js/programSpeakers.js"></script>
+</body>
+
+</html>
```

Comment:
- [ ]Kindly redesign the about page to match that of the [figmal design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
###**Figmal Design**
![about](https://user-images.githubusercontent.com/55596970/181654301-44fb72e7-4ed2-4e64-a331-7310005bfbfb.png)